snmp/assets/monitors/device_unreachable.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "[SNMP] Device unreachable alert on {{snmp_device.name}} in namespace {{device_namespace.name}}",
+  "type": "query alert",
+  "query": "avg(last_5m):max:snmp.device.reachable{*} by {snmp_device,device_namespace,agent_host} < 1",
+  "message": "{{#is_alert}}\nA network device with IP {{snmp_device.name}} in namespace {{device_namespace.name}} is unreachable and can't be monitored anymore.\n{{/is_alert}}\n\n{{#is_alert_recovery}}\nA network device with IP {{snmp_device.name}} in namespace {{device_namespace.name}} is reachable again.\n{{/is_alert_recovery}}\n\nTo know more about the status of your device, you can have more information from the [NDM page for the device {{device_namespace.name}}:{{snmp_device.name}}](/infrastructure/devices/graph?inspectedDevice={{device_namespace.name}}%3A{{snmp_device.name}}).",
```

Comment:
I see there's your mail with the message on the monitor but not here.
Not sure if you want to keep that in the monitor.
<img width="870" alt="image" src="https://user-images.githubusercontent.com/104822957/221159542-ea30cc16-33e0-4f7d-b7e8-d9bf08bba97f.png">

Reply:
Should be fixed thank you!